src/App.tsx

```diff
@@ -33,7 +33,8 @@ function LessonInteractiveArea(lesson: ILesson) {
   const exercisesListItems = [];
   for (const exercise of lesson.exercises) {
     exercisesListItems.push(
-      <Snippet.SnippetFrame {...exercise.content} />
+      <Snippet.Snippet {...exercise.content} />
+      // <Snippet.SnippetFrame {...exercise.content} />
```

Comment:
Do we still need this comment here?

Reply:
Was planning on doing clean pass when this feature is closer to complete